.github/workflows/create-and-publish-docker.yml

```diff
@@ -0,0 +1,66 @@
+
```

Comment:
This is pretty much the same as the previous file just with different triggers and env variables, right? 

Not sure if it's worth it but maybe we could DRY this up using a [reusable workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows)?

Reply:
I think it is worth it (and mentioned as TODO in the introducing description). 
It is definitely planned and I will look into it. Is it okay for you to postpone it a little? I'd create issues for all left over TODOs. First priority was to get the main part reviewed and agreed upon, before over-engineering can start : D

Reply:
Sounds good to me.